common/app/dev/DevAssetsController.scala

```diff
@@ -14,7 +14,13 @@ object DevAssetsController extends Controller with ExecutionContexts {
       if (MimeTypes.isText(mime)) s"${mime}; charset=utf-8" else mime
     }
 
-    val resolved = new File(s"static/hash/$path").toURI.toURL
+    val hashFile = new File(s"static/hash/$path")
+
+    val resolved = if (hashFile.exists()) {
+      hashFile.toURI.toURL
+    } else {
+      new File(s"static/src/$path").toURI.toURL
+    }
```

Comment:
This is needed in order to load the jspm source files via XHR in dev.

/cc @rich-nguyen Note we have swapped the order.